docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`912` Adds total net value to the dashboard balance table.
```

Comment:
Can you also specify the other tables at which the totals are added? These stuff should be in the changelog

Reply:
Aah @kelsos also please switch it from bug to feature. Wasn't there for some time. We can score a feature like that :smile: